docs/resources/Guild.md

```diff
@@ -1008,10 +1008,10 @@ By default, prune will not remove users with roles. You can optionally include s
 
 ###### Query String Params
 
-| Field         | Type                                        | Description                              | Default |
-| ------------- | ------------------------------------------- | ---------------------------------------- | ------- |
-| days          | integer                                     | number of days to count prune for (1-30) | 7       |
-| include_roles | string; comma-delimited array of snowflakes | role(s) to include                       | none    |
+| Field         | Type                                         | Description                                           |
+| ------------- | -------------------------------------------- | ----------------------------------------------------- |
+| days          | integer                                      | number of days to count prune for (1-30), default `7` |
+| include_roles | ?string; comma-delimited array of snowflakes | role(s) to include                                    |
```

Comment:
Same here, can it actually be passed a literal "null" value? Also, wouldn't it be better for kachine readability if `string` was here and `comma seperated ...` was in the description?